src/ethereum/wallet.rs

```diff
@@ -16,6 +16,16 @@ use num::BigUint;
 use num256::Uint256;
 use url::Url;
 
+use anyhow::anyhow;
```

Comment:
:upside_down_face: I prefer it if all the `use` statements are adjacent and the `pub use` statements appear in a separate block.

Reply:
me too :)